lib/Module/IntrinsicCleaner.cpp

```diff
@@ -49,6 +49,58 @@ bool IntrinsicCleanerPass::runOnModule(Module &M) {
   return dirty;
 }
 
+llvm::Constant* IntrinsicCleanerPass::createConstantVector(llvm::LLVMContext& ctx, 
+  llvm::Value* pConstantVector, unsigned mode) { 
+  llvm::Constant* pConstant = NULL;
+  llvm::ConstantInt* pCInt = NULL;
+  Type* pType = NULL;
```

Comment:
```suggestion
  Type* pType = nullptr;
```

Use `nullptr` instead of `NULL`.

Reply:
It should be changed like this